### PR TITLE
Write to ash_history too

### PR DESCRIPTION
**Pull request by alexcb**
- the debugger should produce an .ash_history file incase the user
enters ash instead of bash.

Signed-off-by: Alex Couture-Beil <alex@earthly.dev>

### CODE REVIEW

File: cmd/debugger/main.go [hunk 1]
```diff
@@ -66,12 +66,18 @@ func handleWinChangeData(ptmx *os.File, data []byte) error {
...
-	if err != nil {
-		return err
+	for _, f := range []string{
+		"/root/.ash_history",
+		"/root/.bash_history",
+	} {
+
+		f, err := os.Create(f)
+		if err != nil {
+			return err
```
Comment (vladaionescu) [inline cmd/debugger/main.go:76]: Wonder if we should `continue`?

File: cmd/debugger/main.go [hunk 2]
```diff
@@ -66,12 +66,18 @@ func handleWinChangeData(ptmx *os.File, data []byte) error {
 }
 
 func populateShellHistory(cmd string) error {
-	f, err := os.Create("/root/.bash_history")
-	if err != nil {
-		return err
+	for _, f := range []string{
+		"/root/.ash_history",
```
Comment (vladaionescu) [inline cmd/debugger/main.go:70]: Do we need to also `MkdirAll("/root")`?

**Review by vladaionescu — APPROVED** [review]

File: cmd/debugger/main.go (hunk 2 above)
Comment (alexcb) [reply cmd/debugger/main.go:70]: I don't think so, if that were the case the image would be very broken I suspect.

File: cmd/debugger/main.go (hunk 1 above)
Comment (alexcb) [reply cmd/debugger/main.go]: good point, how about I switch it over to using multierrors?

Comment (vladaionescu) [reply cmd/debugger/main.go]: sounds good

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**